aws-dynamodb.yml

```diff
@@ -15,10 +15,11 @@
 version: 1
 name: csb-aws-dynamodb
 id: bf1db66a-1316-11eb-b959-e73b704ea230
-description: Beta - Amazon DynamoDB
-display_name: Amazon DynamoDB (Beta)
+description: Beta - CSB Amazon DynamoDB
+display_name: CSB Amazon DynamoDB (Beta)
 image_url: https://www.pnglogos.com/images/other/aws-dynamodb.svg
-documentation_url: https://aws.amazon.com/dynamodb/
+documentation_url: https://docs.vmware.com/en/Tanzu-Cloud-Service-Broker-for-AWS/1.1/csb-aws/GUID-reference-aws-dynamodb.html
```

Comment:
should we  set this to v 1.2?
https://docs.vmware.com/en/Tanzu-Cloud-Service-Broker-for-AWS/1.2 will work when this tile becomes available

Reply:
That was the only one that I had made that mistake :)